frontend/awx/resources/groups/hooks/useGroupsFilters.tsx

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react';
```

Comment:
We should add ability to filter by group type
![Screenshot 2024-01-26 at 8 59 30 AM](https://github.com/ansible/ansible-ui/assets/39280967/744812d0-5305-4837-b218-2513804fc53c)